lambdas/handlers/nems_message_handler.py

```diff
@@ -9,10 +11,24 @@
 
 
 @set_request_context_for_logging
+@ensure_environment_variables(
+    names=["APPCONFIG_APPLICATION", "APPCONFIG_CONFIGURATION", "APPCONFIG_ENVIRONMENT"]
+)
 @override_error_check
 @ensure_environment_variables(["LLOYD_GEORGE_DYNAMODB_NAME"])
```

Comment:
There are two '@ensure_environment_variables', can you merge them into one?

Reply:
done